### PR TITLE
Fixed issue #19068: Batch actions dropdown is disabled after first use on User management page

### PR DESCRIPTION
**Pull request by gabrieljenik**

### CODE REVIEW

File: application/extensions/admin/grid/MassiveActionsWidget/assets/listActions.js [hunk 1]
```diff
@@ -369,28 +369,28 @@ $(document).off('pjax:scriptcomplete.listActions').on('pjax:scriptcomplete.listA
 
 
 function switchStatusOfListActions(e) {
-    var checkboxes = $('input[type="checkbox"]');
-    var actionButton = $('.massiveAction');
+    var checkboxSelector = $('#usermanagement--action-toggleAllUsers, .usermanagement--selector-userCheckbox');
```
Comment (Shnoulle) [inline application/extensions/admin/grid/MassiveActionsWidget/assets/listActions.js:372]: Maybe a better selector ? We don't use it on other Batch actions ?

`input[type="checkbox"]` is clearly too large …

File: application/extensions/admin/grid/MassiveActionsWidget/assets/listActions.js [hunk 2]
```diff
@@ -369,28 +369,28 @@ $(document).off('pjax:scriptcomplete.listActions').on('pjax:scriptcomplete.listA
...
-        });
-        return isChecked;
-    }
+// Function to check if at least one checkbox is checked
+function isAnyCheckboxChecked() {
+    var isChecked = false;
+    $('.usermanagement--selector-userCheckbox').each(function () {
+        if ($(this).is(':checked')) {
+            isChecked = true;
+            return false; // Break out of the loop
```
Comment (Shnoulle) [inline application/extensions/admin/grid/MassiveActionsWidget/assets/listActions.js:390]: `$('.usermanagement--selector-userCheckbox').filter(":checked").length`

**Review by Shnoulle — COMMENTED** [review]
1. switchStatusOfListActions is used only by usermanagement ?
2. js need optimization (not related but how … really needed)

**Comment by gabrieljenik** [comment]
Check now please. Thanks

File: application/extensions/admin/grid/MassiveActionsWidget/assets/listActions.js [hunk 3]
```diff
@@ -369,28 +369,31 @@ $(document).off('pjax:scriptcomplete.listActions').on('pjax:scriptcomplete.listA
 
 
 function switchStatusOfListActions(e) {
-    var checkboxes = $('input[type="checkbox"]');
-    var actionButton = $('.massiveAction');
+    var checkboxSelector = '.grid-view-ls input[type="checkbox"]';
```
Comment (Shnoulle) [inline application/extensions/admin/grid/MassiveActionsWidget/assets/listActions.js:372]: 👍 seems great !

File: application/extensions/admin/grid/MassiveActionsWidget/assets/listActions.js [hunk 4]
```diff
@@ -369,28 +369,31 @@ $(document).off('pjax:scriptcomplete.listActions').on('pjax:scriptcomplete.listA
...
     // Attach an onchange event handler to all checkboxes
-    checkboxes.on('change', function () {
+    $(document).on('change', checkboxSelector, function () {
+        // This assumes there is only one massive and one grid in the page.
+        // @todo: 
+        // - Stamp the related grid-id in the massive action button (see massive action widget).
+        // - From checkbox traverse to grid. Fetch grid id.
+        // - Use grid-id to get a more robust link in between grid and massive actions.
```
Comment (Shnoulle) [inline application/extensions/admin/grid/MassiveActionsWidget/assets/listActions.js:379]: 👍

**Review by Shnoulle — APPROVED** [review]
Clean 👍

File: application/views/userManagement/index.php [hunk 5]
```diff
@@ -28,7 +28,10 @@
             'dataProvider' => $model->search(),
             'columns' => $model->getManagementColums(),
             'massiveActionTemplate' => $massiveAction,
-            'lsAfterAjaxUpdate' => ['LS.UserManagement.bindButtons();'],
+            'lsAfterAjaxUpdate' => [
+                'bindListItemclick();',
```
Comment (Shnoulle) [inline application/views/userManagement/index.php:32]: Another issue fixed here ?

**Review by Shnoulle — APPROVED** [review]

**Pull request merged**